docs/javascript/v2/how--to-guides/record-and-live-stream/hls/hls-timed-metadata.mdx

```diff
@@ -42,70 +42,64 @@ start_date and end_date will be in UTC format.
 ### App side implementation
 
 ```javascript
```

Comment:
should this be typescript? does it make a difference though?

Reply:
I don't think so.